common/changes/@microsoft.azure/openapi-validator-rulesets/akhilailla-fix_RPC-Post-V1-09_and_RPC-Get-V1-01_2023-11-02-15-20.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "Update RPC-GET-V1-01(GetOperation200) to allow GET with 202 response code and RPC-Post-V1-09(LroExtension) to exclude GET from x-ms-long-running-operation presence validation",
+      "type": "patch"
```

Comment:
"comment": "Update RPC-GET-V1-01(GetOperation200) to allow GET with 202 response code if the GET represents the location header polling url. Update RPC-Post-V1-09(LroExtension) to exclude GET from x-ms-long-running-operation presence validation",

Reply:
I didnt add
"if the GET represents the location header polling url" cause that isnt being checked in this rule